README.md

```diff
@@ -139,7 +139,9 @@ Based on the type of compute, you have several options for installing the Secret
 
 **To install the Secrets Manager Agent**
 
-1. Use the `install` script provided in the repository\. 
+1. `cp target/release/aws-secrets-manager-agent aws_secretsmanager_agent/configuration/aws-secrets-manager-agent`
+1. `cd aws_secretsmanager_agent/configuration/aws-secrets-manager-agent`
```

Comment:
This is backwards. We don't want people to copy the binary into the source directory where they might accidentally commit it. Instead we want to copy the install script into the build directory, cd into the build directory, and run the install (though ideally we would just have the build put everything in the build directory).

Reply:
Instead of copying stuff around, can we just change the binary path to `../../target/release/aws-secrets-manager-agent` (wherever Cargo outputs the binary)? That should make stuff simpler.

Reply:
Changed the install script to read binary from `../../target/release`.

Reply:
I think Rob was also fixing this here https://github.com/aws/aws-secretsmanager-agent/pull/33, he might just have to rebase on top of this PR once this is merged.

Reply:
I will close https://github.com/aws/aws-secretsmanager-agent/pull/33 and the README can be updated in this PR instead.